### PR TITLE
Add SnowGem extension

### PR DESCRIPTION
**Pull request by Snowgem**
Hello,
We're just implemented SnowGem extension and tested it. Please check and if there's any problem, fell free to tell us.
Thank you so much.

### CODE REVIEW

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/snowgem/SnowgemExtension.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        return null;
+    }
+
+    @Override
+    public IRateSource createRateSource(String sourceLogin) {
+        if (sourceLogin != null && !sourceLogin.trim().isEmpty()){
+            StringTokenizer st = new StringTokenizer(sourceLogin, ":");
+            String exchangeType = st.nextToken();
+            String preferredFiatCurrency = FiatCurrency.USD.getCode();
+            if ("coinmarketcap".equalsIgnoreCase(exchangeType)){
```
Comment (generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/snowgem/SnowgemExtension.java]: Remove Coinmarketcap ratesource.

**Comment by Moktar2k** [comment]
CMC rate source has been removed from the extension.

**Comment by daxicek** [comment]
Hello,

do you need anything more from us? Thank you!

**Comment by rpanak-generalbytes** [comment]
thanks for the commits

**Pull request merged**